### DISCUSSION TITLE
Default routing matrix generation

### DISCUSSION THREAD

**Discussion by KelvinChung2000** [FPGA-Research/FABulous#195] [Ideas]
Provide multiple default routing matrix (sometimes known as switch box) generators so the user can choose which routing matrix pattern to use instead of designing one from scratch. 
[Common switch matrix design](https://www.researchgate.net/figure/Typical-Switch-Box-Configurations_fig2_221224917)

**Comment by Sacoder0000** [comment]
Which one is being used in demo now?

**Reply by KelvinChung2000** [reply]
I am unsure. I think it is a custom design.

**Reply by Sacoder0000** [reply]
> I am unsure. I think it is a custom design.

Is anyone working on it now?

**Reply by KelvinChung2000** [reply]
I am not aware anyone is working on that at the moment.
